src/utils/formHelpers.js

```diff
@@ -0,0 +1,18 @@
+import Joi from "joi";
+
+export function getErrorClass(error) {
+  if (!error) {
+    return "";
```

Comment:
please make this return undefined

Reply:
basically your snapshots should not be saying class="" or class="null", it should just add no class

i thought that returning "" here would prevent that but i didn't look at how it was called